galsim/table.py

```diff
@@ -195,20 +195,20 @@ def __call__(self, x):
             if dimen > 2:
                 raise ValueError("Arrays with dimension larger than 2 not allowed!")
             elif dimen == 2:
-                f = np.empty_like(x.ravel()).astype(float)
-                self.table.interpMany(x.astype(float).ravel(),f)
+                f = np.empty_like(x.ravel(), dtype=float)
```

Comment:
Interesting.  Maybe.  (Although I only get about 25% faster.)  But either is much faster than using `astype`.
```
$ python -m timeit -s 'import numpy; a = numpy.random.random((128,128))' 'x = numpy.empty_like(a, dtype=float)'
1000000 loops, best of 3: 0.908 usec per loop
$ python -m timeit -s 'import numpy; a = numpy.random.random((128,128))' 'x = numpy.empty_like(a)'
1000000 loops, best of 3: 0.677 usec per loop
 python -m timeit -s 'import numpy; a = numpy.random.random((128,128))' 'x = numpy.empty_like(a).astype(float)'
100000 loops, best of 3: 7.7 usec per loop
```